firmware/user/modes/mode_joust_game.c

```diff
@@ -495,8 +495,8 @@ void ICACHE_FLASH_ATTR joustClearWarning(void* arg __attribute__((unused)) )
     joust.instructionTextIdx = OLED_WIDTH;
     joustDrawMenu();
     // Start the timer to scroll text
-    os_timer_arm(&joust.tmr.ScrollInstructions, 34, true);
-    setOledDrawTime(34);
+    os_timer_arm(&joust.tmr.ScrollInstructions, 24, true);
+    setOledDrawTime(24);
```

Comment:
Changing the OLED framerate can cause crashes. You also have the option of scrolling the text faster and not changing the framerate, it'll just be a bit choppier

Reply:
Changed this back to 34, to be safe.